demisto_sdk/__main__.py

```diff
@@ -3546,6 +3546,65 @@ def run_unit_tests(
     sys.exit(unit_test_runner(file_paths, verbose))
 
 
+@main.command(short_help="Setup integration environments")
+@click.option(
+    "-i",
+    "--input",
+    type=PathsParamType(
+        exists=True, resolve_path=True
+    ),  # PathsParamType allows passing a list of paths
+    help="A list of content packs/files to validate.",
+)
+@click.option(
+    "--create-virtualenv",
+    is_flag=True,
+    default=False,
+    help="Create a virtualenv for the environment",
+)
+@click.option(
+    "--overwrite-virtualenv",
+    is_flag=True,
+    default=False,
+    help="Overwrite existing virtualenvs. Use with the create-virtualenv flag",
+)
+@click.option(
+    "--secret-id",
+    help="Secret ID, to use with Google Secret Manager instance with `DEMISTO_SDK_GCP_PROJECT_ID` environment variable set.",
+    required=False,
+)
+@click.option(
+    "--instance-name",
+    required=False,
+    help="Instance name to configure in XSOAR/XSIAM.",
+)
+@click.option("--test-module", required=False, is_flag=True, default=False)
```

Comment:
add help? i believe its when you want to run the test module of the instance to see its success?
call it maybe `run_test_module`